src/components/events/EventView.jsx

```diff
@@ -1,16 +1,131 @@
-import React from 'react'
-import banner from '../../assets/images/bannerevent.png'
+import React from "react";
+
+//styles
+import {
+    banner,
+    top_div,
+    middle_div,
+    bottom_div,
+    container,
+    panel_left,
+    panel_right,
+    space_around,
+    main
+} from "../style_modules/EventView.module.scss";
+
 const EventView = props => {
-    console.log(props)
+
+    //destructure event information passed through props
+    const {
+        id,
+        title,
+        description,
+        start,
+        end,
+        creator,
+        locations,
+        event_images
+    } = props.location.state.item;
+
+    console.log(props.location.state.item);
+
+    //destructure first item in locations array (why is it an array?)
+    const {
+        name,
+        street_address,
+        street_address_2,
+        city,
+        zipcode,
+        state
+    } = locations[0];
+
+    //convert start date to Date object
+    const startDate = new Date(start);
+    const endDate = new Date(end);
+
+    //create string for displaying event time in hours and minutes
+    const startHours = startDate.getHours();
```

Comment:
solid time parsing functions! There is a library in our dependencies called moment.js that gives us some more sophisticated tools for parsing dates, including adjusting the timezones automatically.